interfaces/devdoc/string_utils.md

```diff
@@ -0,0 +1,61 @@
+`string_utils` requirements
+============
```

Comment:
Should do the `#` titles instead of `====`

Reply:
done here and everywhere in this repo